docs/DataProviders.md

```diff
@@ -360,10 +371,25 @@ export default (type, resource, params) => {
             options.method = 'PUT';
             options.body = JSON.stringify(params.data);
             break;
+        case UPDATE_MANY:
+            const query = {
+                filter: JSON.stringify({ id: params.ids }),
+            };
+            url = `${apiUrl}/${resource}?${stringify(query)}`;
+            options.method = 'PATCH';
```

Comment:
Is `PATCH` a better method than `PUT` for an **update many**?

Reply:
https://stackoverflow.com/questions/24241893/rest-api-patch-or-put

Reply:
ok, thanks ;)